tests/generateLibrary/faker/spec.ts

```diff
@@ -0,0 +1,26 @@
+import { anA, seedMocks } from './mocks';
+
+it('should generate dynamic values when dynamicValues is true', () => {
+    const a1 = anA();
+    expect(a1).toMatchSnapshot();
+
+    const a2 = anA();
+    expect(a2).toMatchSnapshot();
```

Comment:
Will this snapshot always be the same when you run the tests multiple times?
As you don't have a `seedMock` it could be dangerous to assume so no?

Reply:
Yes, this snapshot will have the same value even if it is run the test multiple times.
Even if the user does not explicitly call seedMock, the seed will be called once.
https://github.com/ardeois/graphql-codegen-typescript-mock-data/blob/a048815387735e23259004ff39d26ec3f05a00c1/src/mockValueGenerator.ts?plain=1#L81

Testing that a1 and a2 dynamically output different values while being fixed by seeding is useful for VRT and other applications.